sirepo/job_api.py

```diff
@@ -174,6 +174,18 @@ def api_sbatchLogin():
     return _request(_request_content=r)
 
 
+@api_perm.require_user
+def api_statelessCompute():
+    return _request(
+        req_data=PKDict(
+            computeJobHash='unused',
+            report='statelessCompute',
+            **sirepo.http_request.parse_post().req_data
```

Comment:
i think this should be before, not after. The problem is that the external caller could override any value including computeJobHash and report, which are important to not override (well, the report at least).

Reply:
3c17ea4e1